pkg/utils/version.go

```diff
@@ -0,0 +1,50 @@
+package utils
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// GetLatestReleaseVersion fetches the tag of the latest release.
+func GetLatestReleaseVersion(exe Executor) (string, error) {
+
+	ret, err := exe.GH("api", "-H", "Accept: application/vnd.github+json", "-H", "X-GitHub-Api-Version: 2022-11-28", "/repos/elhub/gh-dxp/releases/latest")
```

Comment:
what is the 'ret' var short for here?

Reply:
ret is short for return. good catch, though, I was thinking I should rename this but clearly forgot